packages/rps/config/jest/jest.config.js

```diff
@@ -41,4 +41,5 @@ module.exports = {
     'node',
     'mjs',
   ],
+  roots: ['<rootDir>/src'],
```

Comment:
Why is this necessary?

Reply:
It prevents jest from finding the compiled version of any mocks (source in `/src/`, output in `build`). Without this, it finds both and complains about a duplicate.